pilot/pkg/networking/core/v1alpha3/route/route_test.go

```diff
@@ -128,4 +128,186 @@ func TestBuildHTTPRoutes(t *testing.T) {
 		}
 		g.Expect(routes[0].GetRoute().GetHashPolicy()).To(gomega.ConsistOf(hashPolicy))
 	})
+
+	t.Run("ForVirtualServiceWithSubsetsWithRingHash", func(t *testing.T) {
```

Comment:
Add a test case for a DestinationRule with no `TrafficPolicy` object please; I suspect you'll get some nil-pointer panics due to `rule.GetTrafficPolicy().GetLoadBalancer().GetConsistentHash()`.

Reply:
This will not panic thanks to the accessors